src/bin/bitmaskd.rs

```diff
@@ -27,18 +28,57 @@ use bitmask_core::{
     },
 };
 use log::info;
+use rgbstd::interface::Iface;
+use serde::{Deserialize, Serialize};
 use tokio::fs;
 use tower_http::cors::CorsLayer;
 
-async fn issue(
-    TypedHeader(auth): TypedHeader<Authorization<Bearer>>,
-    Json(issue): Json<IssueRequest>,
-) -> Result<impl IntoResponse, AppError> {
+async fn issue(Json(issue): Json<IssueRequest>) -> Result<impl IntoResponse, AppError> {
     info!("POST /issue {issue:?}");
 
-    let nostr_hex_sk = auth.token();
+    let issuer_keys = save_mnemonic(&get_marketplace_seed().await, "").await?;
+
+    let sk = issuer_keys.private.nostr_prv;
+
+    let issue_res = issue_contract(&sk, issue).await?;
+
+    Ok((StatusCode::OK, Json(issue_res)))
+}
```

Comment:
I'm really worried about the security implications of this... Please secure this with a bearer token, similar to here:

https://github.com/diba-io/bitmask-core/pull/202/files#diff-69dce2abeb3a3dbe31441f8e383f2c9ed664335f5293f0c94fd0ad22ba53c080R255

We can then put the auth token in the server's env vars, and then it can be provided by the server calling it.

Reply:
I'm with you but i'm going to wait until all is working because is much faster to test it like that. So when the hut8 server is working properly i will do another commit with this request.

Reply:
Sounds good